src/c++/vdo/base/recovery-journal.c

```diff
@@ -669,8 +669,8 @@ static int initialize_recovery_block(struct vdo *vdo,
 	/*
 	 * Ensure that a block is large enough to store RECOVERY_JOURNAL_ENTRIES_PER_BLOCK entries.
 	 */
-	STATIC_ASSERT(RECOVERY_JOURNAL_ENTRIES_PER_BLOCK
-		      <= ((VDO_BLOCK_SIZE - sizeof(struct packed_journal_header)) /
+	BUILD_BUG_ON(RECOVERY_JOURNAL_ENTRIES_PER_BLOCK
```

Comment:
Here again, indentation on the next two lines needs to shift one space to the left to fix alignment.